app/src/main/java/com/example/stardewvalleypockettracker/ChecklistActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.example.stardewvalleypockettracker;
+
+import android.os.Bundle;
+import android.view.View;
+
+public class ChecklistActivity extends BaseActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.toolbar);
+        setTitle(R.string.activity_checklists);
+        setSettingsVisibility(View.INVISIBLE);
```

Comment:
Settings should be visible everywhere except the settings activity (it's the default state so this can be removed)